Makefile

```diff
@@ -1953,10 +1953,12 @@ ifdef NEED_ACCESS_ROOT_HANDLER
 	COMPAT_OBJS += compat/access.o
 endif
 
+ifndef NO_PTHREADS
```

Comment:
missing also in the next line (or will break in macOS when `make NO_UNIX_SOCKETS=1`

ifndef NO_UNIX_SOCKETS

Reply:
Good catch. Thanks!

I'm going to try a different approach that will hopefully simplify things.  I'm going to conditionally define the 
FSMONITOR_* symbols in config.mak.uname and then we don't have to worry about the ifdef'ing in the
main Makefile.

```
	# The builtin FSMonitor on MacOS builds upon Simple-IPC.  Both require
	# Unix domain sockets and PThreads.
	ifndef NO_PTHREADS
	ifndef NO_UNIX_SOCKETS
	FSMONITOR_DAEMON_BACKEND = darwin
	FSMONITOR_OS_SETTINGS = darwin
	endif
	endif
```